src/vs/workbench/contrib/codeEditor/electron-sandbox/startDebugTextMate.ts

```diff
@@ -58,26 +51,36 @@ class StartDebugTextMate extends Action {
 		}]);
 	}
 
-	public async run(): Promise<any> {
-		const pathInTemp = join(this._environmentService.tmpDir.fsPath, `vcode-tm-log-${generateUuid()}.txt`);
-		const logger = createRotatingLogger(`tm-log`, pathInTemp, 1024 * 1024 * 30, 1);
-		const model = this._getOrCreateModel();
+	async run(accessor: ServicesAccessor) {
+		const textMateService = accessor.get(ITextMateService);
+		const modelService = accessor.get(IModelService);
+		const editorService = accessor.get(IEditorService);
+		const codeEditorService = accessor.get(ICodeEditorService);
+		const hostService = accessor.get(IHostService);
+		const environmentService = accessor.get(INativeWorkbenchEnvironmentService);
+		const loggerService = accessor.get(ILoggerService);
+		const fileService = accessor.get(IFileService);
+
+		const pathInTemp = joinPath(environmentService.tmpDir, `vcode-tm-log-${generateUuid()}.txt`);
+		await fileService.createFile(pathInTemp);
```

Comment:
@bpasero I do not think this is necessary - Underlying logger will take care of it.

Reply:
Seems to be needed because the file is read right after, but I left @alexdima chime in on that

Reply:
LGTM